Change title 'Create a trial registration? to something like 'Registe…

laraaleite opened the pull request:
…r with Azure Cost Management'.

Change title 'Create a trial registration? to something like 'Register with Azure Cost Management'. 
The current title gives the idea that this will be temporary.
The same feedback applies to the articles for other types of subscriptions:
https://docs.microsoft.com/en-us/azure/cost-management/quick-register-azure-sub
https://docs.microsoft.com/en-us/azure/cost-management/quick-register-csp

azurecla commented:
Hi __@krishmasand__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@krishmasand : Thanks for your contribution to the Azure documentation! The author, @jrowlandjones, has been notified to review your proposed change.

jrowlandjones commented:
#sign-off thank you @krishmasand for this update!

laraaleite committed:
Change title 'Create a trial registration? to something like 'Register with Azure Cost Management'.

Change title 'Create a trial registration? to something like 'Register with Azure Cost Management'. 
The current title gives the idea that this will be temporary.
The same feedback applies to the articles for other types of subscriptions:
https://docs.microsoft.com/en-us/azure/cost-management/quick-register-azure-sub
https://docs.microsoft.com/en-us/azure/cost-management/quick-register-csp
